lib/vendors/axis-communications/sv_vendor_axis_communications.c

```diff
@@ -886,7 +886,7 @@ sv_vendor_axis_communications_set_attestation_report(signed_video_t *sv,
     }
   }
   // Initialize Onvif if private key is set before.
-  if (sv->sign_data && sv->onvif) {
+  if (sv->sign_data->key && sv->onvif) {
```

Comment:
You need to check `sv->sign_data` for not being a NULL pointer as well, otherwise you get a segmentation fault here.